Repositories/MovieRepository.cs

```diff
@@ -79,6 +81,33 @@ public void Delete(int? id)
             }
         }
 
+        public MovieGenreViewModel GetMovieGenreVM(string searchString, string movieGenre)
+        {
+            IQueryable<string> genreQuery;          
+            var movieGenreVM = new MovieGenreViewModel();
+
+            using(var context = Context)
+            {
+                genreQuery = from m in context.Movies orderby m.Genre select m.Genre;
```

Comment:
in this next line "movieGenreVM.genres = new SelectList(genreQuery.Distinct().ToList());"
you can use directly the query